### PR TITLE
OpenSubDiv: bump

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: media-libs/opensubdiv/opensubdiv-3.3.3.recipe [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+and GPU architectures. This codepath is optimized for drawing deforming \
+subdivs with static topology at interactive framerates. The resulting limit \
+surface matches Pixar's Renderman to numerical precision."
+HOMEPAGE="http://graphics.pixar.com/opensubdiv/"
+COPYRIGHT="2013-2018 Pixar"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v${portVersion//\./_}.tar.gz"
+CHECKSUM_SHA256="2dc81b3a085e692cca3166ac88751e4674a9ddf5b5d7935adf677bb2bd3f2d2f"
+SOURCE_DIR="OpenSubdiv-${portVersion//\./_}"
```
Comment (alaviss) [inline media-libs/opensubdiv/opensubdiv-3.3.3.recipe]:
```diff
 CHECKSUM_SHA256="2dc81b3a085e692cca3166ac88751e4674a9ddf5b5d7935adf677bb2bd3f2d2f"
+SOURCE_FILENAME="opensubdiv-$portVersion.tar.gz"
 SOURCE_DIR="OpenSubdiv-${portVersion//\./_}"
```

File: media-libs/opensubdiv/opensubdiv-3.3.3.recipe [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+defineDebugInfoPackage opensubdiv$secondaryArchSuffix \
+	$libDir/libosdCPU.so.$libVersion \
+	$libDir/libosdGPU.so.$libVersion
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	export GLEW_LOCATION="`finddir B_SYSTEM_DEVELOP_DIRECTORY`"
```
Comment (alaviss) [inline media-libs/opensubdiv/opensubdiv-3.3.3.recipe]: Can `portPackageLinksDir` be used instead?

File: media-libs/opensubdiv/patches/opensubdiv-3.3.3.patchset [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+index 3920896..254a4db 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -337,7 +337,7 @@ endif()
+ if(NOT NO_CUDA)
+     find_package(CUDA 4.0)
+ endif()
+-if(NOT NO_GLFW AND NOT NO_OPENGL AND NOT ANDROID AND NOT IOS)
++if(NOT NO_GLFW AND NOT NO_OPENGL AND NOT ANDROID AND NOT IOS AND NOT HAIKU)
+     find_package(GLFW 3.0.0)
```
Comment (alaviss) [inline media-libs/opensubdiv/patches/opensubdiv-3.3.3.patchset]: This shouldn't be a problem if we provides `.cmake` files in glfw package.

Comment (extrowerk) [reply media-libs/opensubdiv/patches/opensubdiv-3.3.3.patchset]: There is no glfw port yet.

**Pull request merged**